src/edu/csus/ecs/pc2/ui/EditProblemPane.java

```diff
@@ -5140,7 +5218,23 @@ private JPanel getCustomValidatorOptionsSubPanel() {
             gbc_labelWhatsThisCLICSValStd.gridx = 2;
             gbc_labelWhatsThisCLICSValStd.gridy = 3;
             customValidatorOptionsSubPanel.add(getLabelWhatsThisCLICSValStd(), gbc_labelWhatsThisCLICSValStd);
-        }
+            
+            // add the "This is an interactive problem" checkbox to the subpanel
```

Comment:
The above comment relates to what I was saying in my comments on class `Problem`.  The comment clearly indicates that the concept is that there should be a _checkbox_ indicating "this is an interactive problem" -- but in fact what the code is adding is a _radio button_.   While it's true that a radio button _can_ be used in a way that it operates the same as a checkbox, THIS radio button is doing way more than that (it is, as I said in the comment on the `Problem` class, overloaded).  

Suppose for example that we accept the concept that the radio button represents the same thing as a "problem is interactive" checkbox.  Then what happens when one of the other radio buttons in the `ButtonGroup` is selected?  The (purported) "problem is interactive" button gets _deselected_.  Does that mean the problem suddenly is NOT interactive?   (Not if I understand the interface as presented.)

As I said elsewhere, I think the concepts of "checkbox indicating the problem is interactive" and "problem uses the CLICS interactive interface" need to be separated.

Having said that, I will concede that this review possibly comes to late to make it feasible to make changes to address this issue... in which case, so be it.  But my process for doing code reviews is, when I see what I consider a flaw, I will point it out.

Reply:
Actually, that comment is wrong.  It's a radio button as you indicate.  IV's can ONLY be (custom)  **CLICS** type validators (at least, that is all we are supporting since that is what we have a spec for).  They can not be PC2 type validators.  Selecting a PC2 type validator correctly means  it is "non-interactive".  There are 3 choices for a  custom validate: **pc2**, **clics**, **clics interactive**.  That is it.  Adding a checkbox next to "_Use CLICS Standard Interface_" makes the interface confusing to use (IMHO).  The radio button makes it crystal clear what you are getting.  And, if  the user wants more explanation, there's always the What's this? question mark.

If this is sticking point, then after the merge an issue can be created discussing the merits of changing the UI.